lib/compress.js

```diff
@@ -2592,9 +2592,10 @@ merge(Compressor.prototype, {
         }
         // x && (y && z)  ==>  x && y && z
         // x || (y || z)  ==>  x || y || z
-        if (self.right instanceof AST_Binary
-            && self.right.operator == self.operator
-            && (self.operator == "&&" || self.operator == "||"))
+        if (self instanceof AST_Binary
+         && self.right instanceof AST_Binary
+         && self.right.operator === self.operator
+         && (self.operator === "&&" || self.operator === "||"))
```

Comment:
It's not my personal preference. I just try to preserve the indent style of project.

Uglify classic multiline `if` condition:
```
if (some_very_long_condition_1
    && some_very_long_condition_2) {
    body;
}
```
I've also seen this in the code base:
```
if (some_very_long_condition_1
    && some_very_long_condition_2)
{
    body;
}
```
either is fine.